server/app/repository/ProgramRepository.java

```diff
@@ -227,6 +235,14 @@ public PaginationResult<Application> getApplicationsForAllProgramVersions(
                 "lifecycle_stage",
                 ImmutableList.of(LifecycleStage.ACTIVE, LifecycleStage.OBSOLETE));
 
+    if (submitTimeFrom.isPresent()) {
+      query = query.where().gt("submit_time", submitTimeFrom.get());
```

Comment:
changed it for the from. most from - to semantics i've encountered are inclusive of from and exclusive of to.

> I'm unsure how this interact with pagination, if we ever expect a flow where someone takes the timestamp of the last result of one call, and passed that in as the start time for the next then we'll need to be a little more careful then.

since we can use ID as a proxy for submit time i don't think this will be an issue even if we allow more flexibility with sort order in the future.